src/System.Runtime.Loader/tests/ReferencedClassLib/ReferencedClassLib.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
+using System.Resources;
+using System.Globalization;
+
+namespace ReferencedClassLib
+{
+    public class Program
+    {
+        static public string Describe(string lang)
+        {
+            try
+            {
+                ResourceManager rm = new ResourceManager(typeof(ReferencedStrings));
```

Comment:
can't you just use the typeof(Program)?

Reply:
this apply to similar places too.

Reply:
I could have if I had named the resource files differently.  I can rename.

Reply:
Or you can just use the other ResourceManager constructor that takes the resource set name.